Sources/ForageSDK/Component/ForagePANTextField/FloatingTextField.swift

```diff
@@ -8,9 +8,9 @@
 
 import UIKit
 
-public class FloatingTextField: UITextField {
+internal class FloatingTextField: UITextField {
```

Comment:
verified that this wasn't being used by the client. We can revisit how we want to expose floating text field customization in the future.